### PR TITLE
fixed

### PR DESCRIPTION
**Pull request by 5thaks07**

### CODE REVIEW

File: controllers/poi.controller.js
Comment (TomStrong) [inline controllers/poi.controller.js:1]: Fixes:
This code won't delete a POI if an admin does it.
There should be a `console.log` in the catch block.
The status code in the catch block should almost always be 500 as it is an unexpected 'internal server error'.

Recommendations: :)
You could also have the success message out side of the `if-else` block then you would only need one.
And try to remember to save manually to format the file :P

**Review by TomStrong — CHANGES_REQUESTED** [review]
Check comments for suggested changes

**Review by TomStrong — APPROVED** [review]
Great, nice work!

**Review by FaizanAhmedSP707 — APPROVED** [review]
Nicely done Shiv!

**Pull request merged**